src/components/layout/DocNav.js

```diff
@@ -31,6 +31,24 @@ class DocNav extends Component {
     }
   }
 
+  handleScroll() {
+    if (document.querySelectorAll('.Collapsible.submenu__item.open')[0] !== undefined) {
```

Comment:
You should invert the test and return early to make the code more readable/less complex (guard clause)